dashboard/views.py

```diff
@@ -625,6 +641,57 @@ def are_weighted_assignments_hidden(course_id, df):
             return True
 
 
```

Comment:
Variable naming could be more consistent and clear in `find_binning_grade_value`. If `item` and `element` are interchangeable, then stick to one?

Reply:
Done